src/core/core.controller.js

```diff
@@ -958,14 +984,18 @@ class Chart {
   unbindEvents() {
     const me = this;
     const listeners = me._listeners;
-    if (!listeners) {
-      return;
-    }
 
     me._listeners = {};
     each(listeners, (listener, type) => {
       me.platform.removeEventListener(me, type, listener);
     });
+
+    if (me._responsiveListeners) {
```

Comment:
If we initialize as `{}` instead of `undefined`, could we avoid this if? might save a few bytes

Reply:
Yes; I'm using whether or not `_responsiveListeners` is set to track when `options.responsive` changes, so initializing as `{}` would interfere with that.

I've addressed the rest of the feedback. Thanks!